src/components/task-plan/homework/add-exercises.cjsx

```diff
@@ -98,6 +98,8 @@ AddExercises = React.createClass
 
     sharedProps =
         exercises: exercises.all
+        watchStore: TaskPlanStore
+        watchEvent: 'change-exercise-'
         onExerciseToggle: @onExerciseToggle
```

Comment:
Can we move the `watch..` props from `sharedProps` and directly onto `ExeriseCard` below?  That will make it more clear that they are `ExerciseCard` specific and not used by `ExerciseDetails`.